.gitignore

```diff
@@ -17,4 +17,7 @@
 # Ignore .DS_store file
 .DS_Store
 
+# Node.js / npm
+node_modules
```

Comment:
I figured committing it after each npm install felt right in this case since it'll allow us to check any updates if necessary. Also means we can keep the deployment pipeline simpler.

Reply:
Happy to review this decision, might be better to avoid having to commit things, just will need to make sure travis is setup correctly.

Reply:
Going to give this a go...